internal/cache/controller_obj_cache.go

```diff
@@ -2785,11 +2775,17 @@ func checkAndSetVRFFromNetwork(client *clients.AviClient) bool {
 		return true
 	}
 
-	networkName := lib.GetNetworkName()
-	if networkName == "" {
-		utils.AviLog.Warnf("Param networkName not specified, skipping fetching of the VRF setting from network")
+	networkList, err := lib.GetVipNetworkList()
+	if err != nil {
+		utils.AviLog.Warnf("Error gettting Network name: %s, skipping fetching of the VRF setting from network", err.Error())
+		return true
+	}
+
+	if len(networkList) == 0 {
+		utils.AviLog.Warnf("Network name net specified, skipping fetching of the VRF setting from network")
 		return true
 	}
+	networkName := networkList[0]
```

Comment:
do we need to check for all networks specified in vipNetworkList. I guess the logic @aaha97 wrote for AviInfraSetting CRD might come in handy here?